src/main/webapp/app/shared/metis/posting-create-edit-modal/post-create-edit-modal/post-create-edit-modal.component.ts

```diff
@@ -209,4 +211,21 @@ export class PostCreateEditModalComponent extends PostingCreateEditModalDirectiv
             courseWideContext: this.posting.courseWideContext,
         };
     }
+
+    private postValidator() {
+        return {
+            // the pattern ensures that the title and content must include at least one non-whitespace character
+            title: [this.posting.title, [Validators.required, Validators.maxLength(TITLE_MAX_LENGTH), Validators.pattern(/^(\n|.)*\S+(\n|.)*$/)]],
```

Comment:
You use this pattern multiple times in this component. Does it make sense to move it into a variable?

Reply:
It is only used once in an inline if statement at line 86. In my opinion it does not make sense "yet"  😄.